src/toil/fileStores/abstractFileStore.py

```diff
@@ -158,36 +161,33 @@ def open(self, job: Job) -> Generator[None, None, None]:
                 self._dumpAccessLogs()
 
     # Functions related to temp files and directories
-    def getLocalTempDir(self):
+    def getLocalTempDir(self) -> str:
         """
         Get a new local temporary directory in which to write files that persist for the duration of
         the job.
 
         :return: The absolute path to a new local temporary directory. This directory will exist
                  for the duration of the job only, and is guaranteed to be deleted once the job
                  terminates, removing all files it contains recursively.
-        :rtype: str
         """
-        return os.path.abspath(tempfile.mkdtemp(dir=self.localTempDir))
+        return os.path.abspath(tempfile.mkdtemp(prefix="t", dir=self.localTempDir))
```

Comment:
Why this change?

Reply:
My bad, I think I accidentally included that. Just removed it.